care/emr/reports/renderer/template_engine.py

```
@@ -0,0 +1,173 @@
+            .replace("(", "")
+            .replace(")", "")
+        )
+        if phone.startswith("+91") and len(phone) >= 13:  # noqa: PLR2004
+            return f"+91 {phone[3:8]} {phone[8:]}"
+        if len(phone) == 10:  # noqa: PLR2004
+            return f"{phone[:5]} {phone[5:]}"
+        return phone
+
+    @staticmethod
+    def _current_date(format_str: str = "%d/%m/%Y") -> str:
+        return datetime.now().strftime(format_str)  # noqa: DTZ005
```

Ruff (0.14.6)

Unused `noqa` directive (non-enabled: `DTZ005`)

Remove unused `noqa` directive

(RUF100)